code/datums/mapgen/Cavegens/IcemoonCaves.dm

```diff
@@ -35,12 +35,15 @@
 	)
 
 /datum/map_generator/cave_generator/icemoon/surface
+	weighted_open_turf_types = list(/turf/open/misc/asteroid/snow/icemoon = 1)
 	flora_spawn_chance = 4
 	weighted_mob_spawn_list = null
-	initial_closed_chance = 53
+	initial_closed_chance = 0
```

Comment:
what's goin on here?

Reply:
all of them on the top z?

Reply:
I am a bit biased because I think it also just looks so much better